mdyd-dev/marketplace-kit
# Deploy zipped assets to s3

Fixes: https://github.com/mdyd-dev/desksnearme/issues/8336

New option for `deploy` command
`marketplace-kit deploy staging --direct-assets-upload`

## QA
You can install cli from this PR branch with `npm install -g mdyd-dev/marketplace-kit#deploy-zipped-assets-to-s3`

- check `deploy --help` if it lists new option
- deploy some instance with assets
- deploy instance with big assets ex. 200MB 
- deploy instance with enormous amount of assets ex. 2GB
- deploy instance without new parameter
